torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -847,5 +853,16 @@ std::vector<std::string> export_opnames(const script::Module& m) {
   return std::vector<std::string>(names.begin(), names.end());
 }
 
+// Thread local floag (only happens in export, i.e. on server side)
+// to control if
+thread_local bool emitBytecodeDefaultInputs =
```

Comment:
Looks like it can be fixed.

Reply:
This value will be changed in the backport function, so directly apply const does not work. Do you have any suggestion?

Reply:
Hmm I see. Probably try suppress this message, otherwise clang-tidy will always complains this line in CI